src/components/search-panel/search-panel.js

```diff
@@ -0,0 +1,219 @@
+import React from 'react';
+import classNames from 'classnames';
+import { AccordionSet, Accordion, FilterAccordionHeader } from '@folio/stripes-components/lib/Accordion';
+import Checkbox from '@folio/stripes-components/lib/Checkbox';
+import { SearchField } from '@folio/stripes-components';
+
+import css from './search-panel.css';
+
+
+export default function SearchPanel() {
+  return (
+    <AccordionSet>
+      <Accordion
+        label="Search"
+        id="ex-0"
+        separator={false}
+        header={FilterAccordionHeader}
+      >
+        <SearchField
+          id="input-search"
+        />
+      </Accordion>
+      <Accordion
+        label="Status"
+        id="ex-1"
+        separator={false}
+        header={FilterAccordionHeader}
+      >
+        <div className={css['nested-container']}>
+          <Checkbox
+            name="checkbox_1"
+            label="Succesful (25)"
+          />
+          <div className={css['nested-container']}>
```

Comment:
Name classes in camelcase and then refer to them in the following way - `css.nestedContainer`. This adds some sort of lint checking.